dms_auto_classification/README.rst

```diff
@@ -45,12 +45,15 @@ Example of filename pattern to use only .txt files: .txt$
 Directory pattern example 1: Documents
 Directory pattern example 1: Documents / Data
```

Comment:
Is this correct? Shouldn't it be `Documents/Data`?

Reply:
Yes, Documents / Data is correcct according to `complete_name` directory field

![ejemplo](https://github.com/OCA/dms/assets/4117568/400a5202-3661-472d-9715-7b7ae6bc1fe6)